### PR TITLE
[MI-3204]: Load testing base setup

### PR DESCRIPTION
**Pull request by ayusht2810**

### CODE REVIEW

File: scripts/load_config.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	if err != nil {
+		panic(err)
+	}
+
+	var config serializers.Config
+	if err := json.Unmarshal(byteValue, &config); err != nil {
+		panic(err)
+	}
+
+	fmt.Println(config.ConnectionConfiguration.ServerURL)
```
Comment (Kshitij-Katiyar) [inline scripts/load_config.go:30]: need for this statement ?

File: config/config.sample.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
+    "ConnectionConfiguration": {
+        "ServerURL": "http://localhost:8065",
+        "WebSocketURL": "ws://localhost:8065",
+        "AdminEmail": "sysadmin@sample.mattermost.com",
```
Comment (Kshitij-Katiyar) [inline config/config.sample.json:4]: Is this structure given by Mike. I think we a separate struct named adminDetails and use that in connection Configuration

File: scripts/load_config.go (hunk 1 above)
Comment (ayusht2810) [reply scripts/load_config.go:30]: there is already a TODO comment above the function to remove it if not needed. also, we have removed this from next PR

File: config/config.sample.json (hunk 2 above)
Comment (ayusht2810) [reply config/config.sample.json:4]: this was taken as a reference from an already existing load testing tool of mattermost https://github.com/mattermost/mattermost-load-test-ng/blob/2f31e615fd9ec7c50c95d562048038da9b067457/config/config.sample.json#L2

File: serializers/config.go [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+	Username string 
+	Password string 
+	Email string
+}
+
+type ChannelsConfiguration struct {
+	DisplayName string 
+	Name string 
+	Type string
+	TeamId string
```
Comment (Kshitij-Katiyar) [inline serializers/config.go:26]:
```suggestion
	TeamID string
```

**Review by Kshitij-Katiyar — APPROVED** [review]

File: config/config.sample.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
+{
+    "ConnectionConfiguration": {
+        "ServerURL": "http://localhost:8065",
+        "WebSocketURL": "ws://localhost:8065",
```
Comment (manojmalik20) [inline config/config.sample.json]: What's the need for this?

**Review by manojmalik20 — APPROVED** [review]

File: serializers/config.go (hunk 3 above)
Comment (ayusht2810) [reply serializers/config.go:26]: this is already fixed in next PR https://github.com/Brightscout/mattermost-load-test-scripts/blob/d609d6a50388f19659a6be582431f4008c7d81a3/config/config.sample.json#L25

File: config/config.sample.json (hunk 4 above)
Comment (ayusht2810) [reply config/config.sample.json]: removed now

**Pull request merged**